src/popup/components/AssetIcon.vue

```diff
@@ -0,0 +1,55 @@
+<template>
```

Comment:
This is basically `ProtocolIcon.vue` but with the added ability to show token icons instead of only coin icons

Reply:
As far as I remember we have separate icons for AE coin and AE protocol. If this is true we need to have 2 separate components: one for asset and one for the protocol.
Please confirm this with @onvisions

Reply:
That is true and is handled in `feat/eth-support`. I'll check it again

Reply:
Much has changed in `feat/eth-support` in regards to protocol/token icons.
This is why I rebased this PR on top of the `feat/eth-support` branch

Reply:
I think it has been discussed already with @martinkaintas some time ago. We decided to keep different versions of the icon for token and for the protocol.